Source/ProgressionSystemRuntime/Public/Data/PSWorldSubsystem.h

```diff
@@ -160,6 +166,14 @@ class PROGRESSIONSYSTEMRUNTIME_API UPSWorldSubsystem : public UWorldSubsystem
 	UPROPERTY(VisibleInstanceOnly, BlueprintReadWrite, Category = "C++", meta = (BlueprintProtected, DisplayName = "Star Dynamic Progress Material"))
 	TObjectPtr<class UMaterialInstanceDynamic> StarDynamicProgressMaterial = nullptr;
 
+	/** Store the material for locked progress material fill for a star actor */
+	UPROPERTY(BlueprintReadWrite, Category = "C++", meta = (BlueprintProtected, DisplayName = "Star locked Progress Material"))
+	TObjectPtr<class UMaterialInstanceDynamic> StarLockedProgressMaterial = nullptr;
```

Comment:
- All 3 material properties are missing `Transient` specifier: they are runtime and should prevent serializing.
- two of them are missing `VisibleInstanceOnly` and `Advanced`, where `Advanced` is new thing I try to set for all Transient-VisibleInstanceOnly properties
- DisplayName = "Star locked Progress Material", where `locked` should start from capital `L`

Expected result:
	UPROPERTY(VisibileInstanceOnly, BlueprintReadWrite, Transient, Advanced, Category = "C++", meta = (BlueprintProtected, DisplayName = "Star Locked Progress Material"))

Reply:
added https://github.com/moinm3uw/ProgressionSystem/pull/55/commits/c6a59bd8d6a3fb1f9cf70dcc0aa1ff2d5af87d1b